std_msgs/Header header                  # time stamp
float64 delta                           # press time difference (in secs)
float64[] press_distribution            # tower button press distribution